python-sdk/src/astro/sql/operators/load_file.py

```diff
@@ -74,6 +77,7 @@ def __init__(
         self.columns_names_capitalization = columns_names_capitalization
         self.enable_native_fallback = enable_native_fallback
         self.load_options = load_options
+        self.pandas_options = pandas_options
```

Comment:
It would be great if we avoided implementation-specific arguments, and had this as part of `load_options`

Reply:
+1

Reply:
Not quite sure but I feel we may need a separate pandas param https://github.com/astronomer/astro-sdk/pull/1466#discussion_r1060364256

Reply:
changed